litestar/response/template.py

```diff
@@ -45,8 +47,12 @@ def __init__(
     ) -> None:
         """Handle the rendering of a given template into a bytes string.
 
+            .. note:: Either ``template_name`` or ``template_str`` must be provided.
+                If both are provided, ``template_str`` will be used.
```

Comment:
should we make them mutually exclusive up-front and raise an error if both are provided?

Reply:
I think it'd be better to raise an error if both are provided as well.